### PR TITLE
Add missing identifier, use pascal case for method

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/visual-basic/language-reference/statements/for-each-next-statement.md [hunk 1]
```diff
@@ -149,10 +149,10 @@ However, this blocking of modification isn't determined by Visual Basic, but rat
 **Modifying Collection Elements.** The <xref:System.Collections.IEnumerator.Current%2A> property of the enumerator object is [ReadOnly](../../../visual-basic/language-reference/modifiers/readonly.md), and it returns a local copy of each collection element. This means that you cannot modify the elements themselves in a `For Each`...`Next` loop. Any modification you make affects only the local copy from `Current` and isn't reflected back into the underlying collection. However, if an element is a reference type, you can modify the members of the instance to which it points. The following example modifies the `BackColor` member of each `thisControl` element. You cannot, however, modify `thisControl` itself.
 
 ```vb
-Sub lightBlueBackground(ByVal thisForm As System.Windows.Forms.Form)
+Sub LightBlueBackground(thisForm As System.Windows.Forms.Form)
     For Each thisControl As System.Windows.Forms.Control In thisForm.Controls
         thisControl.BackColor = System.Drawing.Color.LightBlue
-    Next thisControl
+    Next
```
[the remark this answers is not in the document]
Comment (Youssef1313) [reply docs/visual-basic/language-reference/statements/for-each-next-statement.md]: The `element` is indeed optional after the `Next`. but this change isn't really to show that. It's useless, and most of the code in docs doesn't put anything after `Next`.

**Comment by KathleenDollard** [comment]
@Youssef1313 You make a good point that the element is optional, but this is the one case where I think it's helpful to include it to clarify the syntax notation.  People sometimes struggle with that notation. 

I can see several approaches that would do both:

- Add a comment in the code that the element is optional
- Add a sentence to the following paragraph that the element is optional
- Add a new explanatory paragraph and give the sample twice - once with and once without the element

I tend away from the last, but could support any of these approaches.

**Comment by Youssef1313** [comment]
@KathleenDollard, If it doesn't look good to just remove it, then in my opinion keeping it without any changes is better to avoid the repetition of a piece information that was mentioned already in the start of the article.

**Comment by KathleenDollard** [comment]
@Youssef1313 I don't want to make the change you propose. I believe the presence of `thisControl` is important to clarify the syntax notation. 

But this caught your eye for a reason. Is there a different way you can make that point? Or do you just want to close this issue?

**Comment by Youssef1313** [comment]
> 
> 
> @Youssef1313 I don't want to make the change you propose. I believe the presence of `thisControl` is important to clarify the syntax notation.
> 
> But this caught your eye for a reason. Is there a different way you can make that point? Or do you just want to close this issue?

I may just revert this specific change and keep the other changes.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
In addition to the other comments, @Youssef1313, I thought that the For Each statement could be shortened by taking advantage of type inference.

**Comment by rpetrusha** [comment]
Thanks,, @Youssef1313. I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**